.github/workflows/ci.yml

```diff
@@ -0,0 +1,123 @@
+name: L0 smoke test regression suite
+
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  verilator:
+    name: Build Verilator
+    runs-on: ubuntu-latest
+    env:
+      CCACHE_DIR: "/opt/caliptra-rtl/.cache/"
+      DEBIAN_FRONTEND: "noninteractive"
+
+    steps:
+      - name: Install prerequisities
+        run: |
+          sudo apt -qqy update && sudo apt -qqy --no-install-recommends install \
+            git autoconf automake autotools-dev curl python3 python3-pip \
+            libmpc-dev libmpfr-dev libgmp-dev gawk build-essential bison flex \
+            texinfo gperf libtool patchutils bc zlib1g zlib1g-dev libexpat-dev \
+            ninja-build ccache libfl2 libfl-dev
+
+      - name: Create Cache Timestamp
+        id: cache_timestamp
+        uses: nanzm/get-time-action@v1.1
+        with:
+          format: 'YYYY-MM-DD-HH-mm-ss'
+
+      - name: Setup cache
+        uses: actions/cache@v2
+        timeout-minutes: 3
+        continue-on-error: true
+        with:
+          path: "/opt/caliptra-rtl/.cache/"
+          key: cache_${{ steps.cache_timestamp.outputs.time }}
+          restore-keys: cache_
+
+      - name: Build Verilator
+        run: |
+          git clone https://github.com/verilator/verilator
+          pushd verilator
+            git checkout v5.002
+            autoconf
+            ./configure --prefix=/opt/verilator
+            make -j `nproc`
+            make install
+          popd
+          cd /opt && tar -czvf verilator.tar.gz verilator/
+
+      - name: Store Verilator binaries
+        uses: actions/upload-artifact@v3
+        with:
+          name: verilator
+          path: /opt/*.tar.gz
+
+  tests:
+    name: L0 regression tests
+    runs-on: ubuntu-latest
+    needs: verilator
+
+    strategy:
+      fail-fast: false
+      matrix:
+        test:
+          - name: smoke_test_veer
```

Comment:
Is it possible to use the test-list from [this](https://github.com/chipsalliance/caliptra-rtl/blob/main/src/integration/stimulus/L0_regression.yml) file to avoid duplication?

Reply:
It definitely is. The test matrix can be generated dynamically. I can do that.